### PR TITLE
Refactor/payment UI tweak

### PR DESCRIPTION
**Pull request by kostysh**
This PR implement minor tweaks to the payment UI

### CODE REVIEW

File: src/components/WinPay.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const WinPay = ({ payment, onSuccess }: WinPayProps) => {
             sx={{
               display: 'flex',
               flexDirection: isDesktop ? 'row' : 'column',
-              alignItems: isDesktop ? 'center' : 'stretch',
+              alignItems: isDesktop ? 'top' : 'stretch',
```
Comment (silksil) [inline src/components/WinPay.tsx:53]: you can also do this `alignItems: { xs: "stretch",  lg: "top" }`

File: src/components/AssetSelector.tsx [hunk 2]
```diff
@@ -31,7 +31,8 @@ export const AssetSelector = ({
     const chain = getNetworkInfo(network.chainId);
     return [...chain.contracts.assets.filter((a) => a.currency === payment.currency)];
   }, [network, payment]);
-  const noOptions = useMemo(() => !!asset && options.length === 0, [options, asset]);
+  const notSelected = useMemo(() => options.length > 0 && asset === undefined, [options, asset]);
+  const noOptions = useMemo(() => options.length === 0, [options, asset]);
```
Comment (bobetbat) [inline src/components/AssetSelector.tsx:35]: assets not needed here possibly missed

File: src/hooks/useAccommodationsAndOffers.tsx/index.ts [hunk 3]
```diff
@@ -46,6 +46,11 @@ export const useAccommodationsAndOffers = (props: SearchTypeProps | void) => {
     [data]
   );
 
+  const getAccommodationByHotelId = useCallback(
```
Comment (silksil) [inline src/hooks/useAccommodationsAndOffers.tsx/index.ts:49]: maybe an idea to also rename the `getAccommodationById` so it's clear what the difference is.

I believe one is based on offerId and the other hotelId, right?

File: src/components/AssetSelector.tsx (hunk 2 above)
Comment (kostysh) [reply src/components/AssetSelector.tsx:35]: you are right, really

Comment (kostysh) [reply src/components/AssetSelector.tsx]: fixed

**Review by bobetbat — APPROVED** [review]
LGTM

**Pull request merged**